torch/_inductor/codegen/cuda/cutlass_lib_extensions/gemm_operation_extensions.py

```diff
@@ -0,0 +1,181 @@
+# To be imported only after cutlass_utils.try_import_cutlass() has been called.
+import enum
+
+from cutlass_library import *  # type: ignore[import]  # noqa: F401, F403
+from cutlass_gemm_operation import *  # type: ignore[import]  # noqa: F401, F403
+
+
```

Comment:
The original gemm_operation.py from Cutlass doesn't allow to generate EVT capable fusion code. I had to copy and modify the code of EmitGemmUniversal3xInstance